docs/resources/navigator_block_mobility_groups_copy.md

```diff
@@ -11,6 +27,61 @@ description: |-
 This Terraform resource is used to copy source Data Mobility Group to target on Apex Navigator.
```

Comment:
Can we add more information here ..something like "used in the process of copying data from one storage group (the source) to another storage group (the target) within the APEX Navigator environment"

Reply:
@gokul-srivathsan Sure Updated.